dictionaries/salad-cloud.txt

```diff
@@ -72,3 +72,8 @@ webui
 WORKDIR
 xformers
 xlvaeapp
+getenv
+timeframe
+azcopy
+gsutil
+howtousethesaladterminal
```

Comment:
```suggestion
howtousethesaladterminal
```

Why? This looks like an error that should be fixed, not ignored.

Reply:
My previous commit failed to pass the spellcheck due to these words, so I added them in the dictionary.

Reply:
for this type of thing, I've been renaming the file (I'm guessing this is from a link) to use hyphens between words, and then updating the link accordingly